package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cliqz/adblocker",
-  "version": "0.7.0",
+  "version": "0.7.1",
```

Comment:
Not sure if this one change should be released individually as `0.7.1` or bundled with a previous change which introduced some breaking change (only for the adblocker part and should not affect any other use-case). In which case I would probably bump to `0.8.0`. Would you mind reverting the version change and wait for the release PR to bump version + update CHANGELOG? @eleniS

Reply:
Same with the `ENGINE_VERSION` ?

Reply:
Yes please. We can once for the full release IMO. That should be enough as long as it invalidates clients using a previous version.